x/compliance/keeper/keeper.go

```diff
@@ -93,6 +100,31 @@ func (k Keeper) GetAddressDetails(ctx sdk.Context, address sdk.Address) (*types.
 		return nil, err
 	}
 
+	// Check if issuer exists. If removed, delete verification details from store
```

Comment:
Won't it break a consensus, since this function is getter? In my opinion it can lead to different states at validators

Reply:
Good point, rather filter verifications by issuer's existance.